### PR TITLE
json twig file

### PR DESCRIPTION
**Pull request by saba1995**
adding JSON twig file for the person entity

### CODE REVIEW

File: templates/person/show.json.twig [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        {
+            "@type": "Person",
+            "givenName": "{{ person.firstName}}",
+            "familyName":"{{ person.lastName}}",
+            "sameAs":[
+                "{{ person.wikipediaUrl}}",
+                "{{ person.viafUrl}}"
+            ],
+            "birthDate":"{{person.dob}}",
+            "deathDate":"{{person.dob}}",
```
Comment (joeytakeda) [inline templates/person/show.json.twig:13]: Typo: should be `person.dod`

File: templates/person/show.json.twig [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            "birthPlace":"{{person.cityOfBirth}}",
+            "deathPlace":"{{person.cityOfDeath}}",
+            "@id": "{{ person.id }}"
+        },
+        {% for titleRole in person.getTitleRoles("Author") %}{% set book = titleRole.title %}{
+            "@type": "book",
```
Comment (joeytakeda) [inline templates/person/show.json.twig:19]: Capitalize?

File: templates/person/show.json.twig [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            "@id":"{{book.id}}",
+            "name":"{{book.title}}",
+            "Genre":"{{book.genre}}",
```
Comment (joeytakeda) [inline templates/person/show.json.twig:22]: Should "Genre" be "genre"?

File: templates/person/show.json.twig [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            "familyName":"{{ person.lastName}}",
+            "sameAs":[
+                "{{ person.wikipediaUrl}}",
+                "{{ person.viafUrl}}"
+            ],
+            "birthDate":"{{person.dob}}",
+            "deathDate":"{{person.dob}}",
+            "birthPlace":"{{person.cityOfBirth}}",
+            "deathPlace":"{{person.cityOfDeath}}",
+            "@id": "{{ person.id }}"
```
Comment (joeytakeda) [inline templates/person/show.json.twig:16]: Should the `@id` also be a full URL (like book, firm)?

**Review by joeytakeda — CHANGES_REQUESTED** [review]

File: templates/person/show.json.twig (hunk 4 above)
Comment (saba1995) [reply templates/person/show.json.twig:16]: Yes I changed it I forgot to commit this!

**Comment by ubermichael** [comment]
I guess renaming the master branch to main messed this PR up. I'm sorry about this @saba1995 . Can you create a new one?

**Comment by saba1995** [comment]
Sure. I am not sure why I did not receive any notification for this. 
I will submit a new PR now.

**Pull request closed without merging**